jekyll/_cci2/insights-glossary.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: classic-docs
+title: "Insights metrics glossary"
+short-title: "Insights metrics glossary"
+short-title: "Metric definitions for CircleCI insights"
+version:
+- Cloud
+---
+
+## Overview
+
+This document provides definitions for all the metrics in CircleCI Insights. You
+can review metrics in the Insights dashboard as well as the Insights API.
+
```

Comment:
Can we add links to each section for easy access?

Reply:
I've added a table of contents to the top of the document.